### ISSUE TITLE
Graphviz is missing and required for some of the dask visualizations

### ISSUE THREAD

**Issue by fperez** [pangeo-data/pangeo#74]
Sorry I don't have the full traceback copied, but it should be very easy to reproduce - running the demos at the [PanGeo JupyterHub server](http://pangeo.pydata.org/hub/login) will produce this Dask error when trying to visualize the task DAG:

```
RuntimeError: Drawing dask graphs requires the `graphviz` python library and the `graphviz` system library to be installed.
```

**Comment by mrocklin** [comment]
Yes, that's a valid concern.  I copy-pasted that notebook into this
repository from elsewhere.  We're still trying to figure out how to
minimize the size of the image while also having the functionality we
want.

**Comment by tylere** [comment]
I ran into this as well while running `examples/custom-computations.ipynb`. 

<details>
    <summary>stack trace</summary>
    
    ---------------------------------------------------------------------------
    ModuleNotFoundError                       Traceback (most recent call last)
    /opt/conda/lib/python3.6/site-packages/dask/utils.py in import_required(mod_name, error_msg)
         91     try:
    ---> 92         return import_module(mod_name)
         93     except ImportError:

    /opt/conda/lib/python3.6/importlib/__init__.py in import_module(name, package)
        125             level += 1
    --> 126     return _bootstrap._gcd_import(name[level:], package, level)
        127 

    /opt/conda/lib/python3.6/importlib/_bootstrap.py in _gcd_import(name, package, level)

    /opt/conda/lib/python3.6/importlib/_bootstrap.py in _find_and_load(name, import_)

    /opt/conda/lib/python3.6/importlib/_bootstrap.py in _find_and_load_unlocked(name, import_)

    ModuleNotFoundError: No module named 'graphviz'

    During handling of the above exception, another exception occurred:

    RuntimeError                              Traceback (most recent call last)
    <ipython-input-12-2875082b444a> in <module>()
          9     L = new_L                                   # swap old list for new
         10 
    ---> 11 visualize(*L)

    /opt/conda/lib/python3.6/site-packages/dask/base.py in visualize(*args, **kwargs)
        454     http://dask.pydata.org/en/latest/optimize.html
        455     """
    --> 456     from dask.dot import dot_graph
        457 
        458     filename = kwargs.pop('filename', 'mydask')

    /opt/conda/lib/python3.6/site-packages/dask/dot.py in <module>()
         10 
         11 
    ---> 12 graphviz = import_required("graphviz", "Drawing dask graphs requires the "
         13                                        "`graphviz` python library and the "
         14                                        "`graphviz` system library to be "

    /opt/conda/lib/python3.6/site-packages/dask/utils.py in import_required(mod_name, error_msg)
         92         return import_module(mod_name)
         93     except ImportError:
    ---> 94         raise RuntimeError(error_msg)
         95 
         96 

    RuntimeError: Drawing dask graphs requires the `graphviz` python library and the `graphviz` system library to be installed.

</details>

**Comment by jacobtomlinson** [comment]
Just for reference adding graphviz adds 112MB to the image.

```
IMAGE               CREATED              CREATED BY                                      SIZE                COMMENT
bdb2ccef1e2d        About a minute ago   /bin/sh -c apt-get update -y && apt-get inst…   112MB
```
